database.txt

```diff
@@ -1,38 +0,0 @@
- +-------------+      +-----------------+   +---------------+   +------------+
- | users       |      | preferences     |   | filters       |   | rules      |
```

Comment:
Don't want to keep this file?
It's what was used to generate the image and thus can be handy if there is a need to update it